lib/include/openamp/virtio.h

```diff
@@ -105,10 +105,8 @@ struct virtio_buffer_info {
  */
 struct virtio_vring_info {
 	struct virtqueue *vq;
-	void *va;
+	struct vring_alloc_info info;
```

Comment:
how about just remove vring_alloc_info? and just pass virtio_vring_info to virtqueue_create?

Reply:
yes, i hesitated between both solutions.  Drawback would that either virqueue should include virtio.h or this structure should migrate to virtqueue.h